README.md

```diff
@@ -21,6 +21,7 @@ The following potions are present in this module:
 * Resist Poison - Reduces the effects of poison statuses by 2 when consumed for 15 seconds.
 * Swim Speed - Doubles the user's swim speed for 10 seconds.
 * Walk Speed - Doubles the user's walk speed for 10 seconds.
+* Explosive - Explodes any object it is used on.
```

Comment:
Would it be more technically correct to say the object *using* the potion (in this case the player) triggers an explotion? It seems like the explosion is centered on the player not a selected target block